index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <title>Fork Me! FCC: Test Suite Template</title>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <header id="main">
```

Comment:
Thanks for making such a polite review. You've raised a concern and you've backed it up with an explanation of `why` you requested for the change :+1: 

However, using the word `main` as a value for the `id` attribute is not that bad. Yes, `main` is a name of an element in HTML, yet, you can decide to use the word as a value for either `class` or `id` attribute. 

Although I understand the point you're trying to raise, I personally believe this conversation depends on an individual's opinion. Therefore, it would have been better if this is raised as an **[ OPTIONAL ]** change, not **Required**. :blush: